roles/ceph-osd/tasks/check_devices_auto.yml

```diff
@@ -0,0 +1,45 @@
+---
+- name: check if the device is a partition (autodiscover disks)
+  shell: "echo '/dev/{{ item.key }}' | egrep '/dev/([hsv]d[a-z]{1,2}|cciss/c[0-9]d[0-9]p|nvme[0-9]n[0-9]p)[0-9]{1,2}$'"
+  with_dict: '{{ ansible_devices }}'
+  changed_when: false
+  failed_when: false
+  register: '{{ combined_ispartition_results }}'
```

Comment:
I think we can keep it simple and call it `ispartition_results` since the combined task doesn't exist anymore.